packages/channels/botpress-channel-slack/yarn.lock

```diff
@@ -52,7 +52,7 @@ ajv-keywords@^3.1.0:
   version "3.1.0"
   resolved "https://registry.yarnpkg.com/ajv-keywords/-/ajv-keywords-3.1.0.tgz#ac2b27939c543e95d2c06e7f7f5c27be4aa543be"
 
-ajv@^5.0.0:
+ajv@^5.0.0, ajv@^5.1.0, ajv@^5.3.0:
```

Comment:
@emirotin , are these changes required for this PR?

Reply:
I just think new versions came up?

Reply:
Yeah, they were auto updated when I installed the optional dependencies